pegasus/sites.v3/hourofcode.com/views/course_carousel.haml

```diff
@@ -0,0 +1,46 @@
+- subheading ||= nil
+- company = request.GET["company"]
+- shuffle ||= false
+- items = tutorials.find_with_grade_level(gradelevel)
+- item_count = 3
+
+- if items.count > 0
+  - if shuffle
+    - items = items.to_a.shuffle.to_h
+
+  %hr
+
+  %h3.centered{style: "font-size: 1.5rem; margin-block: 4rem 2rem"}!= heading
+
+  .learnpage{style: "position:relative"}
+    .carousel-wrapper
+      %div.carousel.course-carousel{id: id}
+        .slides
+          - items.each_slice(item_count) do |slice|
+            .slide.clear
+              - slice.each do |key, tutorial|
+                - url = tutorials.launch_url_for(key, request.host_with_port)
+                - url = "#{url}?company=#{company}" if company
+                .action-block-wrapper
+                  .action-block.action-block--one-col.white.flex-space-between
+                    .content-wrapper
+                      %p.overline-two= tutorial[:age]
+                      %h4{style: "font-size: 18px;"}
+                        = tutorial[:name]
+                      %img{src: tutorial[:image].sub("/images/", CDO.code_org_url("/images/fill-480x360/")).sub(".png", ".jpg")}
```

Comment:
Nit: add alt text tag please

Reply:
Thank you for catching this — just updated!